organizational/position_roles.rst

```diff
@@ -0,0 +1,40 @@
+Position Roles 
+========================
+
+Below are the positions that have allowed Hacker's Society function in the past. The
+listed positions are setup to run ACM and IEEE under the umbrella of Hacker's Society.
+The given descriptions are recommendations are able to be changed as the needs of the
+organization change. Along with that additional positions can be added when need.
+
+**President**
+
+- Runs general body meetings
+- Manages the organization ensuring others accomplish specified tasks
+
+**Vice President of ACM**
+
+- Maintain engagement with `ACM <http://www.acm.org/>`_
+- Keep involvment with CS/CE portion of the department
+
+**Vice President of IEEE**
+
+- Maintain engagement with `IEEE <http://www.ieee.org/index.html>`_ 
+- Keep involvment with EE portion of the department
+- Manage hardware library
+
+**Treasurer**
+
+- Approves and keeps record of all financial transactions
+- Creates and approves all funding requests (from USG, CAA, etc.) and budgets
+- Holds the organization credit card
+- Reimburses members
+
+**Public Relations**
+
+- Create and distrubute information within the campus community
+- Contact companies involving events and their involvement
+- Establish sponorship plans for events
+
+**Secretary**
+
+- Keep minutes for general body meetings 
```

Comment:
Perhaps include a bit more here. Organizational things such as keeping up on google drive, mailing lists, or whatever else falls under the purview of secretary.